### ISSUE TITLE
ERROR deprecated: data.GetJSON was deprecated in Hugo v0.123.0 and will be removed in Hugo 0.136.0. use resources .Get or resources.GetRemote with transform.Unmarshal.

### ISSUE THREAD

**Issue by harisheoran** [nunocoracao/blowfish#1757]
**Describe the bug**
Unable to start the website locally due to this error

ERROR deprecated: data.GetJSON was deprecated in Hugo v0.123.0 and will be removed in Hugo 0.136.0. use resources
.Get or resources.GetRemote with transform.Unmarshal.

**To Reproduce**
Execute this command - hugo server

**Expected behavior**
Run the website 

**Desktop (please complete the following information):**
 - OS: Arch Linux
 - Browser: Firefox

**Hugo & Blowfish versions**
Hugo v0.135.0+extended
Blowfish v2.77.1

**Additional context**

**Comment by StandingPadAnimations** [comment]
Do you have any additional themes/shortcodes that use `getJSON`? Blowfish hasn't used `getJSON` in a good while

**Comment by harisheoran** [comment]
No, I am not using additional themes or shortcodes, only from Blowfish documentation.

**Comment by hudsonm62** [comment]
this could probably be closed - guessing not having the issue anymore based on recent workflows https://github.com/harisheoran/harisheoran.github.io/actions/runs/13373070452/job/37345931424

**Comment by hudsonm62** [comment]
hi mate, you are using custom shortcodes - https://github.com/harisheoran/harisheoran.github.io/blob/main/layouts/shortcodes

These should be handled by blowfish, not you

**Comment by hudsonm62** [comment]
Also looking at your workflows build output, the Author error is caused from some of your overrides

- https://github.com/search?q=repo%3Aharisheoran%2Fharisheoran.github.io+Site.Author&type=code
- https://github.com/search?q=repo%3Aharisheoran%2Fharisheoran.github.io%20getJSON&type=code

**Comment by harisheoran** [comment]
ok, it is solved somehow, didn't change anything, thanks
